### DISCUSSION TITLE
get node position by onClick

### DISCUSSION THREAD

**Discussion by amintabrizi** [xyflow/xyflow#2664] [General]
I want to show a box in top of the node that has been clicked, so I need to have exact position of node in the page.
this [API ](https://reactflow.dev/docs/api/nodes/node-options/#position-required)does not give right position of node.
Can somebody help me?

**Comment by moklick** [comment]
Did you try to use the NodeToolbar component https://reactflow.dev/docs/api/nodes/node-toolbar/ ?

**Comment by amintabrizi** [comment]
No I did not, That seems to be all I want :)
